logging.rst

```diff
@@ -308,6 +308,15 @@ option of your handler to ``rotating_file``:
             ),
         ));
 
+Injecting a Logger into a Service
+----------------------------------
+
+To use a logger in your own service, inject it using Dependency Injection
+with ``@logger``. The tag ``monologlog.logger`` tag must be used. If you want to
```

Comment:
I have updated the sentence to mention the tag in combination with a specific channel.